sys/posix/sockets/posix_sockets.c

```diff
@@ -48,11 +48,25 @@
 #include "thread_flags.h"
 #endif
 
+#include "ringbuffer.h"
+
 /* enough to create sockets both with socket() and accept() */
 #define _ACTUAL_SOCKET_POOL_SIZE   (SOCKET_POOL_SIZE + \
                                     (SOCKET_POOL_SIZE * SOCKET_TCP_QUEUE_SIZE))
 #define SOCKET_BLKSIZE             (512)
 
+/**
+ * @brief   Receive buffer, large enough to fit a whole packet with max MTU
+ *
+ * Todo: How to get used MTU size?
+ */
+#define RECV_BUF_SIZE              (1500)
+
+/**
+ * @brief   Peek buffer, large enough to fit a whole packet with max MTU
+ */
+#define PEEK_BUF_SIZE              RECV_BUF_SIZE
```

Comment:
Why is a separate macro needed for this, if it is not configurable?

Reply:
Well I was not entirely sure whether RECV_BUF_SIZE and PEEK_BUF_SIZE should have the same size.